frontend/src/components/Navigator/components/SideNav.css

```diff
@@ -80,3 +80,11 @@
   top: -2.5rem;
   transition: 0.2s ease-in-out;
 }
+
+.smart-hub--navigator-item:first-child .smart-hub--navigator-link-active {
+  border-top-right-radius: 4px;
+}
+
+.smart-hub--navigator-item:last-child .smart-hub--navigator-link-active {
+  border-bottom-right-radius: 4px;
+}
```

Comment:
Fixing slight style issue on the navigator's side nav where the border radius is squared when the first or last element is selected